src/frameworks/CoreServices/src/LaunchServices/constants.c

```diff
@@ -30,12 +30,15 @@ const CFStringRef _kLSASNKey = CFSTR("LSASN");
 
 const CFStringRef kLSItemContentType = CFSTR("LSItemContentType");
 
+struct OpaqueLSSharedFileListItemRef {};
+static const struct OpaqueLSSharedFileListItemRef _insertItemLast = { };
+
 const CFStringRef kLSSharedFileListFavoriteItems = CFSTR("com.apple.LSSharedFileList.FavoriteItems");
 const CFStringRef kLSSharedFileListFavoriteVolumes = CFSTR("com.apple.LSSharedFileList.FavoriteVolumes");
 const CFStringRef kLSSharedFileListGlobalLoginItems = CFSTR("com.apple.LSSharedFileList.GlobalLoginItems");
 // const LSSharedFileListRef kLSSharedFileListItemBeforeFirst;
 const CFStringRef kLSSharedFileListItemHidden = CFSTR("com.apple.LSSharedFileList.ItemIsHidden");
-// const LSSharedFileListItemRef kLSSharedFileListItemLast;
+const LSSharedFileListItemRef kLSSharedFileListItemLast = &_insertItemLast;
```

Comment:
Looking at the symbol location, the `kLSSharedFileList` symbols should actually live in the SharedFileList framework.

For now, let's just move your new symbols into the SharedFileList framework. In a future PR, we will deal with migrating the other symbols into SharedFileList.